helm/beeai-platform/values.yaml

```diff
@@ -304,3 +306,18 @@ externalS3:
   protocol: "https"
   bucket: "beeai-files"
   region: "us-east-1"
+
+docling:
+  resources:
```

Comment:
`resources: {}` is enough here, lets not define any defaults and leave this to the user, similar to our other components